cluster-scope/base/user.openshift.io/groups/octo-ushift-dev/group.yaml

```diff
@@ -7,3 +7,5 @@ users:
     - cooktheryan
     - oglok
     - screeley44
+    - mcliffor
```

Comment:
doesn't seem like this user exists? was this supposed to be `mclifford` ?

Reply:
@HumairAK patched 

good catch